src/main/java/FC/Frame.java

```diff
@@ -427,24 +462,26 @@ private void studyOrResetClicked() throws SQLException {
 
     private void incorrectButtonClicked() {
         currentCard = studyController.getNextToStudy();
-        cardTextArea.setText(currentCard.getTerm());
+        termTextArea.setText(currentCard.getTerm());
     }
 
-    //BUG
     private void correctButtonClicked() {
         studyController.masterCard(currentCard);
         setNumOfCardsStudyMode();
         if (studyController.getNextToStudy() != null) {
             currentCard = studyController.getNextToStudy();
-            cardTextArea.setText(currentCard.getTerm());
+            termTextArea.setText(currentCard.getTerm());
 
         } else {
-            cardTextArea.setText("You finished the deck! Click RESET to start over.");
+            termTextArea.setText("You finished the deck! Click RESET to start over.");
             correctButton.setEnabled(false);
             incorrectButton.setEnabled(false);
             definitionButton.setEnabled(false);
         }
     }
+    private void definitionButtonClicked() {
+        termTextArea.setText("<html>" + currentCard.getDef() + "</html>");
```

Comment:
if term and def are different labels, this needs to be refactored to be defTextArea